reference/forms/types/choice.rst

```diff
@@ -83,40 +94,68 @@ code, while the **key** is what will be shown to the user.
     and will be removed in 3.0. To read about the old API, read an older version of
     the docs.
 
+.. note::
+
+    Pre selected choices will depend on the **data** passed to field and the values of
```

Comment:
In case of doubt, our official reference is the "American English Oxford Dictionary": [preselected](http://www.oxforddictionaries.com/us/definition/american_english/preselect?q=preselected) is correct.